src/coreclr/src/tools/crossgen2/crossgen2/CommandLineOptions.cs

```diff
@@ -146,6 +147,10 @@ public static Command RootCommand()
                 { 
                     Argument = new Argument<int>(() => Environment.ProcessorCount)
                 },
+                new Option(new[] { "--generate-map-file" }, "Generate the map file")
```

Comment:
Same here... name it to just --map? (we had similar names with crossgen1, like /platform_assemblies_paths, which i shortened to /p). It would be nice to have small names for switches